crates/modelardb_server/src/metadata.rs

```diff
@@ -838,7 +767,7 @@ impl MetadataManager {
                      start_time INTEGER, end_time INTEGER, min_value REAL, max_value REAL) STRICT",
                     model_table_metadata.name
                 )
-                .as_str(),
+                    .as_str(),
```

Comment:
Unclear why additional whitespace has been added.

Reply:
Same as [this response](https://github.com/ModelarData/ModelarDB-RS/pull/124#discussion_r1301881092).